server/src/common/script/parser.js

```diff
@@ -0,0 +1,17 @@
+const request = require('request')
+const JSONStream = require('JSONStream')
+const es = require('event-stream')
+
+request({ url: 'https://public.opendatasoft.com/explore/dataset/accidents-corporels-de-la-circulation-en-france/download/?format=json&timezone=Europe/Berlin' })
+  .pipe(JSONStream.parse('*'))
+  .pipe(es.mapSync(function(data) {
+    if (data.fields.coord != null) {
+      var insert = {
+        lat: data.fields.coord[0],
+        lng: data.fields.coord[1],
+        severity: data.fields.grav
+      }
+    }
+    console.log('\n\n')
```

Comment:
I would recommend to add two comments: one at the beginning of the script like: <br> **parser start** and at the end  when everything ends: **parser end**